includes/payment-methods/class-wc-stripe-upe-payment-method.php

```
@@ -137,6 +177,22 @@ public function __construct() {
+	 * Used for backwards compatibility with deprecated properties.
+	 *
+	 * @param string $property The property name.
+	 * @return mixed
+	 */
+	public function __get( $property ) {
+		if ( 'supported_countries' === $property ) {
+			wc_doing_it_wrong( get_class( $this ) . '->supported_countries', 'Use supported_account_countries or supported_billing_countries instead.', '10.8.0' );
+			return $this->supported_account_countries;
+		}
+
+		throw new Exception( 'Property ' . $property . ' is not defined.' );
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '193', column '13'). (undefined)

(MissingImport)